crates/holochain_zome_types/src/dna_def.rs

```diff
@@ -95,48 +55,52 @@ impl DnaDef {
     }
 }
 
+#[cfg(feature = "full-dna-def")]
 impl DnaDef {
     /// Return a Zome
-    pub fn get_zome(&self, zome_name: &ZomeName) -> Result<zome::Zome, DnaError> {
+    pub fn get_zome(&self, zome_name: &ZomeName) -> Result<zome::Zome, ZomeError> {
         self.zomes
             .iter()
             .find(|(name, _)| name == zome_name)
             .cloned()
             .map(|(name, def)| Zome::new(name, def))
-            .ok_or_else(|| DnaError::ZomeNotFound(format!("Zome '{}' not found", &zome_name,)))
+            .ok_or_else(|| ZomeError::ZomeNotFound(format!("Zome '{}' not found", &zome_name,)))
     }
 
     /// Return a Zome, error if not a WasmZome
-    pub fn get_wasm_zome(&self, zome_name: &ZomeName) -> Result<&zome::WasmZome, DnaError> {
+    pub fn get_wasm_zome(&self, zome_name: &ZomeName) -> Result<&zome::WasmZome, ZomeError> {
         self.zomes
             .iter()
             .find(|(name, _)| name == zome_name)
             .map(|(_, def)| def)
-            .ok_or_else(|| DnaError::ZomeNotFound(format!("Zome '{}' not found", &zome_name,)))
+            .ok_or_else(|| ZomeError::ZomeNotFound(format!("Zome '{}' not found", &zome_name,)))
             .and_then(|def| {
                 if let ZomeDef::Wasm(wasm_zome) = def {
                     Ok(wasm_zome)
                 } else {
-                    Err(DnaError::NonWasmZome(zome_name.clone()))
+                    Err(ZomeError::NonWasmZome(zome_name.clone()))
                 }
             })
     }
 
     /// Change the "phenotype" of this DNA -- the UID and properties -- while
     /// leaving the "genotype" of actual DNA code intact
-    pub fn modify_phenotype(&self, uid: Uid, properties: YamlProperties) -> DnaResult<Self> {
+    #[cfg(feature = "full-dna-def")]
```

Comment:
this whole `impl` is conditional... remove this or add it to all the other fns too?